src/metabase/lib/core.cljc

```diff
@@ -301,8 +303,12 @@
   describe-temporal-interval
   describe-relative-datetime
   available-temporal-buckets
+  raw-temporal-bucket
   temporal-bucket
   with-temporal-bucket]
  [lib.util
   normalized-query-type
-  source-table-id])
+  source-table-id]
+ [lib.walk
+  walk
+  walk-stages])
```

Comment:
did you mean to leave these here?

Reply:
personally I find it's useful but yea I guess I don't need it for now. will remove